assets/source/setup-guide/app/views/LandingPageApp.js

```diff
@@ -21,6 +21,16 @@ import {
  */
 import PrelaunchNotice from '../../../components/prelaunch-notice';
 
+const tosHref = 'https://business.pinterest.com/business-terms-of-service/';
```

Comment:
💅 
Looks like Pinterest has an issue when detecting user's browser language. After a few 301 and 302 redirects, it took me to a 404 page - https://business.pinterest.com/zh/zh/zh/zh/business-terms-of-service. Maybe replace it with `'https://business.pinterest.com/en/business-terms-of-service/'`?
 
![Kapture 2021-11-16 at 17 28 24](https://user-images.githubusercontent.com/17420811/141959198-05beb534-93df-4389-9565-e36dcc85840f.gif)

Reply:
I'm able to reproduce it by changing the preferred language in my browser to Simplified Chinese. But https://business.pinterest.com/zh/business-terms-of-service/ works when it's set to English.
Also redirecting works fine for other languages, like Japanese and Polish.

I created a separate issue to track it https://github.com/woocommerce/pinterest-for-woocommerce/issues/264